examples/layer-browser/src/examples/additional-layers.js

```diff
@@ -70,9 +63,16 @@ const ScenegraphLayerExample = {
   props: {
     id: 'scenegraph-layer',
     data: dataSamples.points,
-    sizeScale: 1,
     pickable: true,
-    getPosition: d => [d.COORDINATES[0], d.COORDINATES[1], Math.random() * 10000]
+    sizeScale: 1,
+    getPosition: d => [d.COORDINATES[0], d.COORDINATES[1], Math.random() * 10000],
+    getPitch: d => Math.random() * 360,
+    getRoll: d => Math.random() * 360,
+    getYaw: d => Math.random() * 360,
+    getScale: d => {
+      const s = Math.random() * 3;
+      return [s, s, s];
```

Comment:
Make this something non-uniform? We need to make sure that scaling and roation are applied in the correct order.